templates/store/search.html

```diff
@@ -105,16 +69,40 @@
                 </label>
               </div>
               <div class="list-block__filters--selects-item">
-                <select class="js-series-select" name="series-select">
-                {# The series list has been limited to sane options, full list is here:
-                https://github.com/juju/charmstore/blob/v5-unstable/internal/router/router.go#L33  #}
-                {% for series in ['all', 'precise', 'trusty', 'xenial', 'bionic', 'centos7'] %}
-                  <option value="{{ series }}"
-                    {% if context.current_series == series %}selected="selected"{% endif %}>
-                    {{ full_series(series) }}
-                  </option>
-                {% endfor %}
-              </select>
+                <span class="p-contextual-menu">
+                  <select name="series-select" class="p-contextual-menu__toggle" aria-controls="#menu" aria-expanded="false" aria-haspopup="true">
+                    {# The series list has been limited to sane options, full list is here:
+                      https://github.com/juju/charmstore/blob/v5/internal/series/series.go  #}
+                    {% for series in ['all', 'kubernetes', 'precise', 'trusty', 'xenial', 'bionic', 'centos7'] %}
+                      <option value="{{ series }}" {% if context.current_series == series %}selected="selected"{% endif %}>
+                        {{ series }}
+                      </option>
+                    {% endfor %}
+                  </select>
+                  {% macro series_tag(series) -%}
+                    <a href="{{ current_url_with_query(series=series) }}" class="series-tag series-tag--{{ series }} u-no-margin--bottom">
+                      {{ series }}
```

Comment:
This seems to be outside the for loop above and so will not have access to `series`

Reply:
This uses a template macro which gets passed the series `{% macro series_tag(series) -%}`. Saves having to repeat this code a number of times.